docs/pattern-reorganizing.md

```diff
@@ -31,3 +31,15 @@ You may want to put some space between the numbers just in case you want to furt
 ```
 
 The numbers will not show up when Pattern Lab displays the name of the pattern in the drop-down navigation. They're simply a re-ordering mechanism.
+
+##Re-ordering Pseudo-Patterns
+
+When re-ordering pseudo-patterns the rules are slightly different. The numbers go **after** the tilde sign (`~`) rather than at the beggining of the file name:
```

Comment:
Great stuff! Only thing here is changing "beggining" to "beginning" then we should be good to go.